$NetBSD: patch-al,v 1.3 2012/02/09 10:39:25 he Exp $

Parrot may end up doing usleep(1000000), which is disallowed by SUSv2.

--- src/scheduler.c.orig	2011-10-18 22:44:11.000000000 +0000
+++ src/scheduler.c
@@ -993,7 +993,7 @@ Parrot_cx_schedule_sleep(PARROT_INTERP, 
     Parrot_cx_runloop_wake(interp, interp->scheduler);
 
     /* A more primitive, platform-specific, non-threaded form of sleep. */
-    if (time > 1) {
+    if (time >= 1) {
         /* prevent integer overflow when converting to microseconds
          * and problems in platforms that doesn't allow usleep more
          * than 1 second */
